file-formats/eeec/README.md

```diff
@@ -0,0 +1,6 @@
+# EEEC File Format
+This file format is used by the ADT editor for Event Consumers (EEEC).
```

Comment:
We have got feedback to not list the use case of the object type here. Therefore, I suggest to remove it.
```suggestion
```